Source/BugsnagClient.m

```diff
@@ -206,9 +207,14 @@ @interface BugsnagClient ()
 @property(nonatomic, strong) BugsnagErrorReportApiClient *errorReportApiClient;
 @property(nonatomic, strong, readwrite) BugsnagSessionTracker *sessionTracker;
 @property (nonatomic, strong) BSGOutOfMemoryWatchdog *oomWatchdog;
+@property (nonatomic, strong) BugsnagPluginClient *pluginClient;
 @property (nonatomic) BOOL appCrashedLastLaunch;
 @end
 
+@interface BugsnagConfiguration ()
+@property(nonatomic, readwrite, strong) NSMutableSet *plugins;
```

Comment:
Are we confident that plugin load ordering won't be an issue?  If we later find that it is this could be replaced with an ordered set implementation.

Reply:
Currently bugsnag-cocoa only uses plugins for cocos2dx - that won't be used at the same time as react-native. In future we could update the set implementation or make the plugins resilient to handling different load orders.